CMake: installation of pdb files with ninja generator & Visual Studio is broken

When you use Ninja generator & Visual Studio, installation of pdb files is broken:

```
**********************************************************************
** Visual Studio 2019 Developer Command Prompt v16.10.4
** Copyright (c) 2021 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'
[0/1] Install the project...
-- Install configuration: "Debug"
-- Installing: C:/Users/spaceim/.conan/data/tidy-html5/5.8.0/_/_/package/4144424e75eb292877f55084b3c3523c9e0ae02c/lib/tidy_staticd.lib
-- Installing: C:/Users/spaceim/.conan/data/tidy-html5/5.8.0/_/_/package/4144424e75eb292877f55084b3c3523c9e0ae02c/include/tidyplatform.h
-- Installing: C:/Users/spaceim/.conan/data/tidy-html5/5.8.0/_/_/package/4144424e75eb292877f55084b3c3523c9e0ae02c/include/tidy.h
-- Installing: C:/Users/spaceim/.conan/data/tidy-html5/5.8.0/_/_/package/4144424e75eb292877f55084b3c3523c9e0ae02c/include/tidyenum.h
-- Installing: C:/Users/spaceim/.conan/data/tidy-html5/5.8.0/_/_/package/4144424e75eb292877f55084b3c3523c9e0ae02c/include/tidybuffio.h
CMake Error at source_subfolder/cmake_install.cmake:50 (file):
  file INSTALL cannot find
  "C:/Users/spaceim/.conan/data/tidy-html5/5.8.0/_/_/build/4144424e75eb292877f55084b3c3523c9e0ae02c/build_subfolder/source_subfolder/tidy-static.dir/Debug/tidy-static.pdb":
  File exists.
Call Stack (most recent call first):
  cmake_install.cmake:37 (include)
```

Here is the pb: https://github.com/htacg/tidy-html5/blob/1ca37471b48a3498f985509828cb3cf85ea129f8/CMakeLists.txt#L419-L423
Too fragile. It assumes multi configuration generator.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -416,11 +416,6 @@ install(TARGETS ${name}
         LIBRARY DESTINATION ${LIB_INSTALL_DIR}
         )
 install( FILES ${HFILES} DESTINATION ${INCLUDE_INSTALL_DIR} )
-if(MSVC)
-    # install(FILES $<TARGET_PDB_FILE:${name}> DESTINATION lib OPTIONAL)
-    INSTALL(FILES ${PROJECT_BINARY_DIR}/${name}.dir/Debug/${name}.pdb
-    DESTINATION lib CONFIGURATIONS Debug )
-endif()
 
 #------------------------------------------------------------------------
 # Dynamic Library
@@ -445,6 +440,9 @@ if (BUILD_SHARED_LIB)
         ARCHIVE DESTINATION ${LIB_INSTALL_DIR}
         LIBRARY DESTINATION ${LIB_INSTALL_DIR}
         )
+    if(MSVC)
+        install(FILES $<TARGET_PDB_FILE:tidy-share> DESTINATION ${LIB_INSTALL_DIR} OPTIONAL)
+    endif()
     if (TIDY_CONSOLE_SHARED) # user wants shared/dll linkage
         list ( APPEND add_LIBS ${name} )
     endif ()    

